setup.py

```diff
@@ -1,22 +1,36 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import setuptools
+from setuptools import setup, find_packages
+from os.path import join as opj, dirname
+import re
 
-#
-# Grab some info from supporting files
-#
-with open('README.md', 'r') as fh:
-    long_description = fh.read()
+version_file = 'htcrystalball/_version.py'
+readme = opj(dirname(__file__), 'README.md')
+mo = re.search(
```

Comment:
What is this searching for?

Reply:
It searches for the value of the `__version__` string in the `_version.py` file. Having a `_version.py` file makes it accessible from the code, rather than only in `setup.py`, which is advantageous if you need to inform the user of the package version (e.g., `htcb --version` returns the `__version___` value from `_version.py`).